aten/src/ATen/native/DispatchStub.h

```diff
@@ -51,6 +51,7 @@ enum class CPUCapability {
 #else
   AVX2 = 1,
   AVX512 = 2,
+  NEON = 3,
```

Comment:
This should be guarded under appropriate ifdef, shouldn't it?

Reply:
Noted, fixed in the next commit.